Lang/Spanish.pj.Lang

```diff
@@ -234,6 +234,7 @@
 #465#  "Mostrar velocidad"
 #466#  "Velocidad de la pantalla:"
 #467#  "Comprobar si Project64 ya está en ejecución"
+#468#  "Directorio Único de Guardados de Juego"

```

Comment:
I would add a noun instead. "Guardados" is mostly used as an adjective.

Suggestion: Analogously to "save states" in English, you could go for "estados guardados".

Also, as opposed to English, Spanish does not capitalize words as much ("Únicos" instead of "únicos", "guardados" instead of "Guardados", "juego" instead of "Juego").